### PR TITLE
Frontier Pass

### PR DESCRIPTION
**Pull request by DizzyEggg**
90% Team go go go

### CODE REVIEW

File: src/frontier_pass.c [hunk 1]
```diff
@@ -1,8 +1,525 @@
...
+};
+
+struct
+{
+    s16 yStart;
+    s16 yEnd;
+    s16 xStart;
+    s16 xEnd;
+}
+static const sPassAreasLayout[] =
```
[the remark this answers is not in the document]
Comment (DizzyEggg) [reply src/frontier_pass.c]: Diego pls, I declared a struct right above it.

[the remark this answers is not in the document]
Comment (Diegoisawesome) [reply src/frontier_pass.c]: Ah yeah whoops, too used to qualifiers coming before the type

[the remark this answers is not in the document]
Comment (DizzyEggg) [reply src/frontier_pass.c]: Yeah, I'm used to them too. But the above syntax is useful for one-time only structs so I don't have to think of a name for them :D

[the remark this answers is not in the document]
Comment (aaaaaa123456789) [reply src/frontier_pass.c]: You can still do `static const struct`...

**Pull request merged**